### ISSUE TITLE
Outlook Desktop, keyboard focus is lost after opening a new inspector window with a pinned add-in

### ISSUE THREAD

**Issue by DmitryKostochko** [OfficeDev/office-js#1162]
We are developing an Outlook add-in for compose mode that supports the Pin/Unpin functionality. In Outlook Desktop, the keyboard focus is lost after creating a new email or replying to an existing email if an add-in is pinned.

## Expected Behavior
The keyboard focus shall remain in the To field in case of creating a new email and in the Body field in case of Reply/ReplyAll/Forward.

## Current Behavior
The keyboard focus is lost, the user needs to click in a certain field in the inspector window to start typing.

## Steps to Reproduce, or Live Example
Create a new empty add-in for compose mode.

New email case:
1. In Outlook Desktop, create a new email.
2. Start the add-in, pin it and close the new email window.
3. Create a new email again. Notice that the keyboard focus is lost, and you cannot type in the To field.

Reply case:
1. In Outlook Desktop, go to File -> Options -> Mail, check the "Open replies and forwards in a new window" option and click OK.
2. Select an email in your Inbox, click Reply.
3. Start the add-in, pin it and close the reply email window.
4. Select an email in your Inbox, click Reply again.
5. Note that the keyboard focus is lost, and you cannot type in the Body field.

## Your Environment
* Platform [PC desktop, Mac, iOS, Office Online]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Version 2004 (build 12730.20250)
* Operating System: Windows 10
* Browser (if using Office Online): ______

**Comment by exextoc** [comment]
I have reproduced the issue on our end, and have logged a bug on the backlog of the product team. I cannot give an ETA to when or if the issue will be addressed.

**Comment by DmitryKostochko** [comment]
Thank you. Please let me know if (or when) you need my help in testing.

**Comment by exextoc** [comment]
Internal tracking id: Office:4155408
